<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>913</width>
    <height>482</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>DataLogger</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QToolButton" name="connectBtn">
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>55</height>
           </size>
          </property>
          <property name="text">
           <string>Connect</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="resetAxisBtn">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>55</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Reset
axis</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="resetSeriesBtn">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>55</height>
           </size>
          </property>
          <property name="text">
           <string>Reset
series</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="saveBtn">
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>55</height>
           </size>
          </property>
          <property name="text">
           <string>Save as...</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="voltageLbl">
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Voltage: 0.00 V</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="currentLbl">
              <property name="text">
               <string>Current: 0.00 mA</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="tempLbl">
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Temp:    0.00 Â°C</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="powerLbl">
              <property name="text">
               <string>Power:   0.00 mW</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="layout">
      <property name="spacing">
       <number>0</number>
      </property>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="tcp_status_label">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Please click 'Connect' button</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="dateTimeLbl">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>12:00:00 | 2017-02-10</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
